src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -1926,6 +1926,24 @@ def test_process_use_stmts_with_accessibility_statements(parser):
     assert symtab.lookup("some_var").visibility == Symbol.Visibility.PUBLIC
 
 
+def test_intrinsic_use_stmt(parser):
+    ''' Tests that intrinsic value is set correctly for an intrinsic module
+    use statement.'''
+    processor = Fparser2Reader()
+    reader = FortranStringReader('''
+        module test
+            use, intrinsic :: ieee_arithmetic, only: isnan =>ieee_is_nan
+            use mymod
```

Comment:
For completeness you can also test the `use, non_intrinsic :: ...` import  which is the only other possible value and it should be the same as default here (no need to reproduce it in the backend, as it is equivalent to no attribute)

Reply:
Added